# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/gfxVRExternal.cpp
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/vr/gfxVRExternal.cpp
--- a/gfx/vr/gfxVRExternal.cpp	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/vr/gfxVRExternal.cpp	Sun May 26 14:30:14 2019 +0000
@@ -8,8 +8,8 @@
 
 #include "prlink.h"
 #include "prenv.h"
-#include "gfxPrefs.h"
 #include "mozilla/Preferences.h"
+#include "mozilla/StaticPrefs.h"
 
 #include "mozilla/gfx/Quaternion.h"
 
@@ -526,7 +526,7 @@
 
 #elif defined(XP_WIN)
   if (mShmemFile == NULL) {
-    if (gfxPrefs::VRProcessEnabled()) {
+    if (StaticPrefs::VRProcessEnabled()) {
       mShmemFile =
           CreateFileMappingA(INVALID_HANDLE_VALUE, NULL, PAGE_READWRITE, 0,
                              sizeof(VRExternalShmem), kShmemName);
@@ -624,11 +624,11 @@
     VRExternalShmem* aAPIShmem /* = nullptr*/) {
   MOZ_ASSERT(NS_IsMainThread());
 
-  if (!gfxPrefs::VREnabled()) {
+  if (!StaticPrefs::VREnabled()) {
     return nullptr;
   }
 
-  if ((!gfxPrefs::VRExternalEnabled() && aAPIShmem == nullptr)
+  if ((!StaticPrefs::VRExternalEnabled() && aAPIShmem == nullptr)
 #if defined(XP_WIN)
       || !XRE_IsGPUProcess()
 #endif